OWNERS

```diff
@@ -4,10 +4,12 @@ reviewers:
   - dberkov
   - jefftree
   - tallclair
+  - ipochi
 approvers:
   - anfernee
   - cheftako
   - jkh52
+  - ipochi
```

Comment:
K/K [policy](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-2) is that approvers have to have first spent 3 months (a release) as a reviewer before becoming an approver.